server/src/server/app.ts

```diff
@@ -1,18 +1,21 @@
+require('dotenv').config();
 import express from "express";
 import bodyParser from "body-parser";
 import cors from "cors";
 import { initializeApplication } from "./server";
+const { photosRouter } = require("./routes/photos");
 
 export const app = express();
 
 app.use(bodyParser.json());
+app.use('/photos', photosRouter);
```

Comment:
Move router below CORS.
Node's `app.use` is a middleware - each request goes over the middlewares in a sequential order - meaning right now your request passed bodyParser, router and then (maybe) cors. It should be bodyParser, cors and then router.